bindings/python/crocoddyl/core/action-base.hpp

```diff
@@ -54,6 +54,27 @@ class ActionModelAbstract_wrap : public ActionModelAbstract, public bp::wrapper<
   }
 
   boost::shared_ptr<ActionDataAbstract> default_createData() { return this->ActionModelAbstract::createData(); }
+
+  void quasiStatic(const boost::shared_ptr<ActionDataAbstract>& data, Eigen::Ref<Eigen::VectorXd> u,
+                   const Eigen::Ref<const Eigen::VectorXd>& x, const std::size_t maxiter, const double tol) {
+    if (boost::python::override createData = this->get_override("quasiStatic")) {
```

Comment:
```suggestion
    if (boost::python::override quasiStatic = this->get_override("quasiStatic")) {
```

Reply:
Thanks Rohan for noticing this detail.

Done!